cmd/podman/main.go

```diff
@@ -65,18 +65,23 @@ var cmdsNotRequiringRootless = map[*cobra.Command]bool{
 	_exportCommand:  true,
 	//// `info` must be executed in an user namespace.
 	//// If this change, please also update libpod.refreshRootless()
-	_loginCommand:   true,
-	_logoutCommand:  true,
-	_mountCommand:   true,
-	_killCommand:    true,
-	_pauseCommand:   true,
-	_restartCommand: true,
-	_runCommand:     true,
-	_unpauseCommand: true,
-	_searchCommand:  true,
-	_statsCommand:   true,
-	_stopCommand:    true,
-	_topCommand:     true,
+	_loginCommand:    true,
+	_logoutCommand:   true,
+	_mountCommand:    true,
+	_killCommand:     true,
+	_pauseCommand:    true,
+	_podKillCommand:  true,
+	_podRmCommand:    true,
+	_podStatsCommand: true,
+	_podStopCommand:  true,
+	_restartCommand:  true,
+	_rmCommand:       true,
+	_runCommand:      true,
+	_unpauseCommand:  true,
+	_searchCommand:   true,
+	_statsCommand:    true,
+	_stopCommand:     true,
+	_topCommand:      true,
```

Comment:
Did you intend for these changes or are they perhaps an unexpected merge dance?  I thought some of @baude's work had moved these elsewhere....

Reply:
they are intended.  It is a list of commands that don't need to get automatically a userns created